process/project-lifecycle.md

```diff
@@ -174,7 +159,20 @@ To archive a project:
 * The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
 * A vote must be finalized with 2/3 approval from the TAC or parent WG.
 
-#### Submission Forms to Archive
 
-1. [Proposal](TBD)
-2. [Legal](TBD)
+## Submission Process
+
+### Project creation or change of lifecycle stage
+
+For initiating the creation of a new project or for requesting a change of a project's lifecycle stage, an application must be submitted to the TAC. To this end, a maintainer of the project (or another interested party if no maintainer is available) creates a PR in this repository with the following changes:
```

Comment:
"(or another interested party if no maintainer is available)"
Was this to accommodate Archived? If so, should we specify so it's not misinterpreted as non-maintainers could push a project from Incubation to Graduation without the maintainers' involvement?

Reply:
Yes, it is. Typically when a project is being archived there is no longer any maintainer around to do any such work so I thought we should provide for the possibility of having someone else. One possibility is to make this a community architect/LF staff member if you're concerned that this might be misused (although I can't imagine this being practically possible if the project is actively maintained).

Reply:
Yes, @annabellegoth2boss, that was the intention and I can change the text to be more explicit. Another alternative used in LF projec is that a member of the TAC opens the PR to archive a project if no maintainer can be found to do this on behave of  the project.

Reply:
```suggestion
For initiating the creation of a new project or for requesting a change of a project's lifecycle stage, an application must be submitted to the TAC. To this end, a maintainer of the project (or any OpenSSF member in the case of Archived projects) creates a PR in this repository with the following changes:
```